src/backend/distributed/citus--7.0-8--7.0-9.sql

```diff
@@ -0,0 +1,73 @@
+/* citus-7.0-8--7.0-9 */
+
+SET search_path = 'pg_catalog';
+
+ALTER TABLE pg_dist_node ADD COLUMN nodecluster text NOT NULL DEFAULT 'default';
```

Comment:
should we use varchar(...) here to prevent names that are too long?

or, might make sense to test what happens if you put in long names (either through INSERT or master_add_node).